mibidata/mibi_image.py

```diff
@@ -261,10 +261,11 @@ def channel_inds(self, channels):
         except (ValueError, AttributeError):
             if self.targets is None:
                 error_msg = f'Channels were indexed with targets only (no ' \
-                    f'masses were given), available targets are {channels}'
+                    f'masses were given), available targets are ' \
+                    f'{self._channels}'
```

Comment:
Sorry if I missed this, but what is the purpose of changing from `channels` to `self._channels` here?

Reply:
This is a separate bug that I noticed. 

`self._channles` are channels available in the mibiimage, `channels` are channels that the function should return.

Reply:
The way it was written made sense to me, so maybe I was misinterpreting it all along. The new error message added here, as you have written it, makes sense. But `Subset of channels, targets or massses not found matching ...` makes sense with the channels that were passes as an argument and not the full list of channels in the image.